src/cls/rbd/cls_rbd.h

```diff
@@ -219,6 +219,9 @@ struct cls_rbd_snap {
     if (parent_overlap) {
       f->dump_unsigned("parent_overlap", *parent_overlap);
     }
+    f->open_object_section("namespace");
+    snapshot_namespace.dump(f);
+    f->close_section();
```

Comment:
I would place this between `protection_status` and `child_count` to preserve existing (decoding) order.

Reply:
And add `timestamp` while at it:

`f->dump_stream("timestamp") << timestamp;`

Reply:
Thanks for the comment, I've fixed it.